### PR TITLE
Removed scrollbar of texarea in  SuccessTip page on FF

### PR DESCRIPTION
**Pull request by stanislav-slavov**

### CODE REVIEW

**Review by davidyuk — COMMENTED** [review]
Screenshot 2020-12-30 at 22 43 21

The scrollbar disappeared, but the text is still cropped, let's use div instead of textarea.

File: src/popup/router/components/Textarea.vue [hunk 1]
```diff
@@ -49,6 +49,7 @@ export default {
   margin-left: auto;
   margin-right: auto;
   word-break: break-word;
+  scrollbar-width: none;
```
Comment (davidyuk) [inline src/popup/router/components/Textarea.vue:52]: Also, we shouldn't disable scrollbars in all textareas.

**Review by Liubov-crypto — COMMENTED** [review]
After logging in I got this issue: 

web wallet error

and this: 
FF error

**Comment by Liubov-crypto** [comment]
Tested, now LGTM

test tip

File: src/popup/router/pages/SuccessTip.vue [hunk 2]
```diff
@@ -116,4 +116,14 @@ export default {
   font-weight: normal;
   margin: 8px 0;
 }
+
+.note {
+  color: $text-color;
+  font-size: $font-size;
+  min-height: 100px;
```
Comment (davidyuk) [inline src/popup/router/pages/SuccessTip.vue:123]: This is not needed because divs are auto-sized.

**Review by davidyuk — APPROVED** [review]

**Review by mradkov — APPROVED** [review]

**Pull request merged**